libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/ElementLogoAtom.kt

```diff
@@ -66,22 +57,35 @@ fun ElementLogoAtom(
             .border(size.borderWidth, borderColor, RoundedCornerShape(size.cornerRadius)),
         contentAlignment = Alignment.Center,
     ) {
-        Box(
-            Modifier
-                .size(size.outerSize)
-                .shapeShadow(
-                    color = shadowColor,
-                    cornerRadius = size.cornerRadius,
-                    blurRadius = size.shadowRadius,
-                    offsetY = 8.dp,
-                )
-        )
+        if (canUseBlurMaskFilter()) {
```

Comment:
Maybe pass this as a parameter, defaulting `canUseBlurMaskFilter()` to be able to preview the composable when this value is false.